settings/deno.vim

```diff
@@ -21,3 +21,57 @@ augroup vim_lsp_settings_deno
       \ }
 augroup END
 
+function! s:handle_deno_status(ctx, server, type, data) abort
+    let l:path = 'deno:/status.md'
+    let l:buffer = bufnr(l:path)
+
+    let l:mods = has_key(a:ctx, 'mods') ? a:ctx['mods'] : ''
+    if l:mods ==# '' && &modified && !&hidden && l:buffer != bufnr('%')
+        let l:mods = &splitbelow ? 'rightbelow' : 'leftabove'
+    endif
+    if l:mods ==# ''
+        if l:buffer == bufnr('%')
+            let l:cmd = ''
+        else
+            let l:cmd = (l:buffer !=# -1 ? 'b ' . l:buffer : 'edit ' . fnameescape(l:path))
+        endif
+    else
+        let l:cmd = l:mods . ' ' . (l:buffer !=# -1 ? 'sb ' . l:buffer : 'split ' . fnameescape(l:path))
+    endif
+
+    let l:text = a:data['response']['result']
+    execute 'silent ' l:cmd
+
+    setlocal modifiable
+    call setline(1, split(l:text, "\n"))
+    setlocal noswapfile buftype=nofile
+    setlocal nomodified
+    setlocal nomodifiable
+endfunction
+
+function! s:status() abort
+  let l:has_extension = 1
+  let l:command_id = lsp#_new_command()
+  let l:ctx = {
+        \ 'counter': l:has_extension,
```

Comment:
What does this `l:has_extension` mean?

Reply:
oops, sorry.
My experimental code(I'll send Goto defenition PR later) were mixed.
Replace to `l:ctxt = {}`